kubernetes/README.md

```diff
@@ -76,6 +76,37 @@ These defaults can be modified using:
 --set cloudserver.autoscaling.config.targetCPUUtilizationPercentage=...
 ```
 
+Pod Disruption Budgets
+----------------------
+
+Zenko relies on several stateful services require a quorum to function properly
+as highly available services to provide resilency to many types of outage
+scenarios. Pod disruption budgets allow for configuration of how many pods of a
+given service can safely fail and continue to operate normally before Kubernetes
```

Comment:
managed by a given statefulset?

Reply:
I mean each statefulset is providing a HA service. Even in Kubernetes terminology using "service" makes sense since that's how processes communicate with each other. Plus allows for easier comprehension for people not well versed in the terminology.